walker_test fails

**Describe the bug**

Running jlaffaye/ftp tests in a build on a Debian system (invoking `cd obj-x86_64-linux-gnu && go test -vet=off -v -p 8 -short github.com/jlaffaye/ftp`), the walker_test fails in `TestEmptyStackReturnsFalse` and `TestCurAndStackSetCorrectly`:

```
...
=== RUN   TestSkipDirIsCorrectlySet
--- PASS: TestSkipDirIsCorrectlySet (0.00s)
=== RUN   TestNoDescendDoesNotAddToStack
--- PASS: TestNoDescendDoesNotAddToStack (0.00s)
=== RUN   TestEmptyStackReturnsFalse
    walker_test.go:107: 
        	Error Trace:	walker_test.go:107
        	Error:      	Expected value not to be nil.
        	Test:       	TestEmptyStackReturnsFalse
--- FAIL: TestEmptyStackReturnsFalse (0.00s)
=== RUN   TestCurAndStackSetCorrectly
    walker_test.go:139: 
        	Error Trace:	walker_test.go:139
        	Error:      	Expected value not to be nil.
        	Test:       	TestCurAndStackSetCorrectly
--- FAIL: TestCurAndStackSetCorrectly (0.00s)
=== RUN   TestCurInit
--- PASS: TestCurInit (0.00s)
```

Does it mean a tighter dependency on base Go modules/API or some other module needs to be declared?


**To Reproduce**
```
go test -vet=off -v -p 8 -short github.com/jlaffaye/ftp
```

**Versions**
- jlaffaye/ftp 0.0~git20190828.6aaa91c
- golang-go 1.15
- stretchr-testify 1.6.1

--- a/conn_test.go
+++ b/conn_test.go
@@ -328,7 +328,7 @@ func openConn(t *testing.T, addr string, options ...DialOption) (*ftpMock, *Serv
 
 // Helper to close a client connected to a mock server
 func closeConn(t *testing.T, mock *ftpMock, c *ServerConn, commands []string) {
-	expected := []string{"FEAT", "USER", "PASS", "TYPE", "OPTS"}
+	expected := []string{"USER", "PASS", "FEAT", "TYPE", "OPTS"}
 	expected = append(expected, commands...)
 	expected = append(expected, "QUIT")
 

--- a/walker_test.go
+++ b/walker_test.go
@@ -104,11 +104,11 @@ func TestEmptyStackReturnsFalse(t *testing.T) {
 	assert, require := assert.New(t), require.New(t)
 
 	mock, err := newFtpMock(t, "127.0.0.1")
-	require.NotNil(err)
+	require.Nil(err)
 	defer mock.Close()
 
 	c, cErr := Connect(mock.Addr())
-	require.NotNil(cErr)
+	require.Nil(cErr)
 
 	w := c.Walk("/root")
 
@@ -136,11 +136,11 @@ func TestCurAndStackSetCorrectly(t *testing.T) {
 	assert, require := assert.New(t), require.New(t)
 
 	mock, err := newFtpMock(t, "127.0.0.1")
-	require.NotNil(err)
+	require.Nil(err)
 	defer mock.Close()
 
 	c, cErr := Connect(mock.Addr())
-	require.NotNil(cErr)
+	require.Nil(cErr)
 
 	w := c.Walk("/root")
 	w.cur = &item{

